### DISCUSSION TITLE
Will there be Phi-SoSerious-Mini trained on Phi-3.5-mini?

### DISCUSSION THREAD

**Discussion by tororon1231** [LostRuins/koboldcpp#1086] [General]
Hi, I really like the model and would like to ask if there will be a new version of the model trained on Phi-3.5-mini?

**Comment by LostRuins** [comment]
Hmm, I'll consider it, but fighting microsoft censorship is an uphill battle and I'm not sure if it's worth it - from what I heard this model is even more censored.

**Reply by tororon1231** [reply]
Well, it has its negatives, but the benchmarks of the new model look good, it is multilingual and supports 128k context.

**Reply by tororon1231** [reply]
@LostRuins You could finetune on top of [Phi-3.5-mini-instruct_Uncensored](https://huggingface.co/SicariusSicariiStuff/Phi-3.5-mini-instruct_Uncensored), which removes a lot of censorship from the model.
